docs/data-access/data-download.md

```diff
@@ -0,0 +1,89 @@
+---
+id: data-download
+title: How to Download Data
+---
+
+The ARGO Data Platform uses the score-cient as a file download manager. Score facilitates the transfer of data with resumable download and built in BAM/CRAM slicing to make data download transfer fast and smooth.
+
+Please note:
+
+- Downloads are done in part and can be paused/resumed as needed
```

Comment:
should the bullets be capitalized? they weren't on the daco page for "... DACO access, please ensure that you"

suggestion:
Downloads are done in `parts` and can be paused and resumed as needed